source/iostreams.tex

```diff
@@ -626,6 +626,7 @@
 \synopsis{Header \tcode{<ios>} synopsis}
```

Comment:
@tkoeppe: I sympathize, but that issue is not the subject of this pull request. Feel free to vent your anger in #566.